Part_1/Calculator/app/src/main/java/com/ensf409/calculator/Calculator.java

```diff
@@ -0,0 +1,223 @@
+package com.ensf409.calculator;
+import java.text.DecimalFormat;
+import java.util.LinkedList;
+
+public class Calculator {
+	private LinkedList<Double> numbers;//holds all numbers in order for current calculation
+	private LinkedList<Character> operations;//holds all operations in order for current calculation
+	private Double currentNumber;//current number being built
+	private double lastSum;//holds value of the lastSum calculated
+	private int afterDec; //counts how many elements past decimal, 0 if no decimal entered
+	private String toDisplay;//string to be returned on ButtonPressed call
+	private char lastReceived;//keeps track of last received char on a ButtonPressed call
+	private boolean divZeroErr;//becomes true if a calculation attempts to divide by zero
+	
+	//initializes calculator
+	public Calculator(){
+		clear();
+		lastSum = 0;
+		lastReceived = 'C';
+		toDisplay = "";
+		divZeroErr = false;
+	}
+	//clears Lists of numbers & operations, sets afterDec and currentNumber to 0
+	//gets called by when 'C' or '=' is pressed and when initializing calculator
+	private void clear(){
+		numbers = new LinkedList<Double>();
+		operations = new LinkedList<Character>();
+		currentNumber = 0.0;
+		afterDec = 0;
+	}
+	//called from '=', calculates sum of stored numbers with respective operations and stores value in lastSum
+	//if a divide by zero is attempted, sets the divZeroErr to true 
+	private void calcSum(){
```

Comment:
I'm not really sure about the name of this function, since the term sum implies that it's going to be adding. I would suggest `calculate`